examples/evaluation/evaluate_with_llm_judge.py

```diff
@@ -0,0 +1,124 @@
+# os.environ["OPENAI_API_KEY"] = "test"
+
+assert "OPENAI_API_KEY" in os.environ, "Please set the OPENAI_API_KEY environment variable."
+
+
+# testing with OpenAI gpt-3.5-turbo
+example = EvaluationExample(
+    input="What is MLflow?",
+    output="MLflow is an open-source platform for managing machine "
+    "learning workflows, including experiment tracking, model packaging, "
+    "versioning, and deployment, simplifying the ML lifecycle.",
+    score=4,
+    justification="The definition effectively explains what MLflow is "
+    "its purpose, and its developer. It could be more concise for a 5-score.",
+    variables={
+        "ground_truth": "MLflow is an open-source platform for managing "
+        "the end-to-end machine learning (ML) lifecycle. It was developed by Databricks, "
+        "a company that specializes in big data and machine learning solutions. MLflow is "
+        "designed to address the challenges that data scientists and machine learning "
+        "engineers face when developing, training, and deploying machine learning models."
+    },
+)
+
+correctness = make_genai_metric(
+    name="correctness",
+    version="v1",
+    definition="Correctness refers to how well the generated output matches "
+    "or aligns with the reference or ground truth text that is considered "
+    "accurate and appropriate for the given input. The ground truth serves as "
+    "a benchmark against which the provided output is compared to determine the "
+    "level of accuracy and fidelity.",
+    grading_prompt="Correctness: If the answer correctly answer the question, below are the "
+    "details for different scores: "
+    "- Score 0: the answer is completely incorrect, doesn’t mention anything about "
+    "the question or is completely contrary to the correct answer. "
+    "- Score 1: the answer provides some relevance to the question and answer one aspect "
+    "of the question correctly. "
+    "- Score 2: the answer mostly answer the question but is missing or hallucinating on one "
+    "critical aspect. "
+    "- Score 4: the answer correctly answer the question and not missing any major aspect",
+    examples=[example],
+    model="openai:/gpt-3.5-turbo-16k",
+    variables=["ground_truth"],
+    parameters={"temperature": 0.0},
+    greater_is_better=True,
+    aggregations=["mean", "variance", "p90"],
+)
+
+eval_df = pd.DataFrame(
+    {
+        "input": [
+            "What is MLflow?",
+            "What is Spark?",
+            "What is Python?",
+            # "What is Delta Lake?",
+            # "What is Apache Iceberg?",
+            # "What is AWS?",
+            # "What is Kubernetes?",
+            # "What is Docker?",
+            # "What is Hadoop?",
+            # "What is TensorFlow?",
+            # "What is PyTorch?",
+            # "What is NoSQL?",
+            # "What is GraphQL?",
+            # "What is REST API?",
+            # "What is Git?",
+            # "What is CI/CD?",
```

Comment:
Lets remove the commented questions out so this can actually be merged with master in future?